src/operator/nn/mkldnn/mkldnn_fully_connected.cc

```diff
@@ -82,6 +82,101 @@ inline static mkldnn::inner_product_backward_weights::primitive_desc GetIPBwdWei
   }
 }
 
+class MKLDNNFullyConnectForward {
```

Comment:
could you take the caching of FullyConnected forward out and submit another PR for this?
my concern is that this PR is for caching backward. mixing it with all other backward operators might cause confusing later. thanks

Reply:
OK. I will move FullyConnected forward out this PR.

Reply:
FullyConnected forward PR see #11611.